packages/www/pages/docs/guides/debugging-guide.mdx

```diff
@@ -0,0 +1,72 @@
+import DocsLayout from "../../../components/DocsLayout";
+
+<DocsLayout>
+
+
+# Debugging Livepeer.com Stream Playback
+
+This step-by-step guide will help you debug common stream playback problems. Try these steps before reaching out to Livepeer.com directly.
+
+We will use the [Livepeer.com test player](/app/test-player) to debug. You must be logged into the Livepeer.com dashboard. 
+
+#### Prerequisite: Locate your playback URL. 
+
+Your playback URL is specific to the stream ingest region. 
+- This [Livepeer.com API guide](/docs/guides/api/playback-a-live-stream) touches on finding your playback URL by combining a base URL with a `playbackId`. If you don't know your stream `playbackId`, you can retrieve it via [`GET /stream/<id>`](/docs/api/stream/get-stream).
+- You can also find the playback URL on the Livepeer.com dashboard stream page, `https://livepeer.com/app/stream/<stream-id>`. Learn more [here](/docs/guides/dashboard/playback-a-stream).
+
+## Step 1
+
+Load your playback URL in the [Livepeer.com test player](/app/test-player). If you get a "Stream not found" error, confirm that your URL is correct. It should look something like this:  https://region-cdn.livepeer.com/hls/123456abcdef7890/index.m3u8
+
+## Step 2
+
+Does anything playback in the player of *your* choice?
+
+- **No:** Go to the next step.
+- **I don't know. I haven't tested in my own player.**: Go to step 4.
+- **Yes, but the quality is poor:** Go to step 5.
+
+## Step 3
+
+Does anything playback in the [Livepeer.com test players](/app/test-player)?
+
+- **Yes, the quality is great:** There may be something wrong with your player configuration. Start by checking if your player is loading the correct playback URL.
+- **No:** Go to the next step.
+- **Yes, but the quality is poor:** Go to step 5
+
+## Step 4
+
+Check the stream status below the Livepeer.com test player. Is it active?
+
+- **No:** No video is being ingested. Confirm your or your streamer’s broadcast software RTMP configuration is set up properly. If you are using OBS to stream into Livepeer, check out steps 4-5 in [this blog post](/blog/first-livepeer-stream-in-five-minutes). 
+- **Yes:** Go to the next step.
+
+## Step 5
+
+Check if you are transcoding. Is there only one `profile` listed next to `Playback settings` below the Livepeer.com test player? 
+
+- **Yes:** You are not transcoding your source stream, and only the source profile is available for playback (no ABR) via the Livepeer.com CDN. There may be something wrong with your source stream. Confirm the keyframe interval is set to 2 in your broadcasting software. Try switching the broadcast software internet provider.
+<br />Or, depending on the resolution, FPS and bitrate of your source and the bandwidth of your internet, playback may be slow and choppy. Consider transcoding your ingested video. If no transcoding renditions are available for playback, you are not making use of ABR.
```

Comment:
What about the "FPS and bitrate of your source" could make the playback choppy? Would be good to include info. I'd be wondering that if I was reading this.

How would they "transcode your ingested video"? i.e. create a new stream and add additional profiles to their stream object, because as things stand, they have none?

Reply:
"If your resolution, FPS and/or bitrate are super high and you have shitty internet bandwidth, playback will be slow and choppy." I'll update with something like this.

I'll also change "Consider transcoding your ingested video" to "Consider transcoding your livestreams."